store/tikv/oracle/oracles/pd_test.go

```diff
@@ -11,29 +11,100 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package oracles_test
+package oracles
```

Comment:
do not rename the packages.

Reply:
Thanks for your review, I agree with you.